template.go

```diff
@@ -68,6 +68,14 @@ func add(arg1, arg2 int) int {
 	return arg1 + arg2
 }
 
+func isTrue(bool string) bool {
+  b, err := strconv.ParseBool(strings.ToLower(bool))
```

Comment:
`strings.ToLower` is not needed because `ParseBool` already handles 't' and 'true'.